images/plbase/python-requirements.txt

```diff
@@ -1,21 +1,21 @@
 ansi2html==1.9.1
-black==24.1.1
+black==24.2.0
 chevron==0.14.0
-coloraide==2.16
-Faker==22.6.0
+coloraide==3.1
```

Comment:
Major version bump here was mainly because internal-use functions got removed: https://github.com/facelessuser/coloraide/releases/tag/3.0

IMO very unlikely to break anything on our end.